manifesto.tex

```diff
@@ -107,37 +107,37 @@ \subsection{Operational Decentralisation}
 
 \subsection{Node Software and Protocol Specification}
 
 Authoring the node software is a difficult task requiring both a huge investment of time and substantial expertise of the underlying protocol. Because of this, most blockchain networks have only one piece of node software. Worse still, networks tend to define their \emph{protocol} in terms of the primary (or only) node software, including all bugs, opinionation, accidental design flaws and implementation details. Bitcoin is a good example of this where its original author provided no protocol definition beyond the codebase and the protocol continues to be defined by that ever-evolving codebase under the dictatorship of a few so-called ``core developers'', unelected and formally unaccountable. In this case, a decision over the codebase implies a decision over the protocol.
 
 As with most software projects, there is generally a single team behind the node software. In highly commercial blockchains, the software is subject to a closed development model in which a company's CEO has ultimate and absolute control. This makes the blockchain laughably compromisable and entirely unfit for use.
 
-However, even in highly open blockchain projects with open-source software (OSS) development models, decisions in software development must still be made and they will tend to happen under the benevolent dictator model as written about in the seminal work by Raymond\cite{cathedral_and_bazaar}. This can lead to a problem where a minority, including said dictator, are of one opinion and the majority are of another. If both sides feel strongly and cannot be reconsiled in a technical manner, then OSS projects may schism and become ``forked'', leading a single project and community to be split into two. The resultant projects and their teams fight for users on the merits of their relevant decision. Typically one project eventually wins out but co-existence and even cooperation is not entirely unheardof.
+However, even in highly open blockchain projects with open-source software (OSS) development models, decisions in software development must still be made and they will tend to happen under the benevolent dictator model as written about in the seminal work by Raymond\cite{cathedral_and_bazaar}. This can lead to a problem where a minority, including said dictator, are of one opinion and the majority are of another. If both sides feel strongly and cannot be reconciled in a technical manner, then OSS projects may schism and become ``forked'', leading a single project and community to be split into two. The resultant projects and their teams fight for users on the merits of their relevant decision. Typically one project eventually wins out but co-existence and even cooperation is not entirely unheard of.
```

Comment:
```suggestion
However, even in highly open blockchain projects with open-source software (OSS) development models, decisions in software development must still be made and they will tend to happen under the benevolent dictator model as written about in the seminal work by Raymond\cite{cathedral_and_bazaar}. This can lead to a problem where a minority, including said dictator, are of one opinion and the majority are of another. If both sides feel strongly and cannot be reconciled in a technical manner, then OSS projects may schism and become ``forked'', leading a single project and community to be split into two. The resultant projects and their teams fight for users on the merits of their relevant decision. Typically one project eventually wins out but co-existence and even cooperation is not entirely unheard-of.
```

Reply:
I think the unhyphenated variant [should also be correct](https://www.collinsdictionary.com/dictionary/english/unheard-of) and is used more often? So I'll leave this for Gav to decide.